config/config.go

```diff
@@ -461,6 +465,15 @@ func Init() error {
 		Parameters.SyncMode = SyncMode
 	}
 
+	if Parameters.SyncStateMaxPeer == 0 {
+		syncStateMaxPeer := uint64(float64(memory.TotalMemory() / defaultSyncStateThreadMemory))
```

Comment:
Why not using `uint32()` instead of `uint64` here?

Reply:
Fixed